### PR TITLE
Update Backblaze template

### PR DESCRIPTION
**Pull request by smccloud**
Add some new variables (DISABLE_AUTOUPDATE, FORCE_LATEST_UPDATE, USER_ID, GROUP_ID, & CLEAN_TMP_DIR), update DISPLAY_WIDTH & DISPLAY_HEIGHT to new defaults.

### CODE REVIEW

File: templates/backblaze.xml [hunk 1]
```diff
@@ -26,11 +26,16 @@ This container needs additional installation steps after running the docker, whi
...
   <Config Name="Timezone" Target="TZ" Default="Etc/UTC" Mode="" Description="Timezone of the container. Timezone can also be set by mapping /etc/localtime between the host and the container.&#9;" Type="Variable" Display="always" Required="false" Mask="false">Etc/UTC</Config>
-  <Config Name="Display Width" Target="DISPLAY_WIDTH" Default="657" Mode="" Description="Width (in pixels) of the virtual screen's window." Type="Variable" Display="always" Required="false" Mask="false">657</Config>
-  <Config Name="Display Height" Target="DISPLAY_HEIGHT" Default="473" Mode="" Description="Height (in pixels) of the virtual screen's window." Type="Variable" Display="always" Required="false" Mask="false">473</Config>
+  <Config Name="Display Width" Target="DISPLAY_WIDTH" Default="900" Mode="" Description="Width (in pixels) of the virtual screen's window." Type="Variable" Display="always" Required="false" Mask="false">657</Config>
```
Comment (nwithan8) [inline templates/backblaze.xml:29]:
```suggestion
  <Config Name="Display Width" Target="DISPLAY_WIDTH" Default="900" Mode="" Description="Width (in pixels) of the virtual screen's window." Type="Variable" Display="always" Required="false" Mask="false">900</Config>
```

File: templates/backblaze.xml [hunk 2]
```diff
@@ -26,11 +26,16 @@ This container needs additional installation steps after running the docker, whi
...
+  <Config Name="Display Height" Target="DISPLAY_HEIGHT" Default="700" Mode="" Description="Height (in pixels) of the virtual screen's window." Type="Variable" Display="always" Required="false" Mask="false">473</Config>
```
Comment (nwithan8) [inline templates/backblaze.xml:30]:
```suggestion
  <Config Name="Display Height" Target="DISPLAY_HEIGHT" Default="700" Mode="" Description="Height (in pixels) of the virtual screen's window." Type="Variable" Display="always" Required="false" Mask="false">700</Config>
```

File: templates/backblaze.xml [hunk 3]
```diff
@@ -26,11 +26,16 @@ This container needs additional installation steps after running the docker, whi
...
   <Config Name="VNC Password" Target="VNC_PASSWORD" Default="" Mode="" Description="Password needed to connect to the application's GUI. See the VNC Password section in the README for more details." Type="Variable" Display="always" Required="false" Mask="true"/>
   <Config Name="X11VNC_EXTRA_OPTS" Target="X11VNC_EXTRA_OPTS" Default="" Mode="" Description="Extra options to pass to the x11vnc server running in the Docker container. WARNING: For advanced users. Do not use unless you know what you are doing." Type="Variable" Display="advanced" Required="false" Mask="false"/>
   <Config Name="ENABLE_CJK_FONT" Target="ENABLE_CJK_FONT" Default="" Mode="" Description="When set to 1, open-source computer font WenQuanYi Zen Hei is installed. This font contains a large range of Chinese/Japanese/Korean characters.&#9;" Type="Variable" Display="advanced" Required="false" Mask="false"/>
```
Comment (nwithan8) [inline templates/backblaze.xml:33]:
```suggestion
  <Config Name="ENABLE_CJK_FONT" Target="ENABLE_CJK_FONT" Default="" Mode="" Description="When set to 1, open-source computer font WenQuanYi Zen Hei is installed. This font contains a large range of Chinese/Japanese/Korean characters." Type="Variable" Display="advanced" Required="false" Mask="false"/>
```

File: templates/backblaze.xml [hunk 4]
```diff
@@ -26,11 +26,16 @@ This container needs additional installation steps after running the docker, whi
...
+  <Config Name="User ID" Tareget="USER_ID" Default="99" Description="When mounting docker-volumes, permission issues can arise between the docker host and the container. You can pass the User_ID permissions to the container with this variable." Type="Variable" Display="always" Required="true" Mask="false">99</Config>
+  <Config Name="Group ID" Target="GROUP_ID" Default="100" Description="When mounting docker-volumes, permission issues can arise between the docker host and the container. You can pass the Group_ID permissions to the container with this variable." Type="Variable" Display="always" Required="true" Mask="false">100</Config>
+  <Config Name="Clean Temp Directory" Target="CLEAN_TMP_DIR" Default="1" Description="When set to 1, all files in the /tmp directory are deleted during the container startup." Type="Variable" Display="always" Required="true" Mask="false">1</Config>
+  <Config Name="Disable Autoupdate" Target="DISABLE_AUTOUPDATE" Default="false" Description="Disables the auto-update of the backblaze client to the latest known-good version at the time of the docker version release."  Type="Variable" Display="always" Required="true" Mask="false">true</Config>
```
Comment (nwithan8) [inline templates/backblaze.xml:40]:
```suggestion
  <Config Name="Disable Autoupdate" Target="DISABLE_AUTOUPDATE" Default="false" Description="Disables the auto-update of the backblaze client to the latest known-good version at the time of the docker version release."  Type="Variable" Display="always" Required="true" Mask="false">false</Config>
```

Should match the `Default` value.

**Review by nwithan8 — CHANGES_REQUESTED** [review]

**Review by nwithan8 — APPROVED** [review]

**Review by Roxedus — APPROVED** [review]

**Pull request merged**